source/connections-survive-step-down/tests/README.rst

```diff
@@ -178,3 +169,9 @@ a successful insert operation does not directly test functionality introduced in
 straightforward way to test driver resiliency against a live replica set undergoing an election. This testing
 methodology is in contrast to the one adopted by the SDAM spec tests that rely entirely on mocking with no actual
 server communication.
+
+
+.. _CMAP: https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst
+.. _PoolClearedEvent: https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#events
```

Comment:
Can you change these to relative links so they work in branches?
```
.. _CMAP: /source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst
.. _PoolClearedEvent: /source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#events
```

Reply:
fixed